README.md

```diff
@@ -347,11 +357,21 @@ sliced "packet".
 
 Fields:
 
- - `enabled`: true/false
  - `average_size`: size in bytes of an average packet
  - `size_variation`: variation in bytes of an average packet (should be smaller than average_size)
  - `delay`: time in microseconds to delay each packet by
 
+### Writing custom toxics
+
+Writing your own toxics for Toxiproxy can be done by copying [toxiproxy.go](https://github.com/Shopify/toxiproxy/blob/master/cmd/toxiproxy.go)
+into a new project and registering your toxic with the server. This will allow you to build your own Toxiproxy
+binary without having to make a full fork of the project. If you think your toxics will be useful to others,
+contribute them back with a Pull Request.
```

Comment:
Right now this is the only way to write custom toxics without merging upstream. It's an alternative to forking the whole project, which is what would have had to be done previously. I'll move this section into the `CREATING_TOXICS.md` file and just link to it, rather than having this weird summary.